### PR TITLE
feat(docs): FRMW-155-Migrate progressbar docs in README to ngdocs

### PR DESCRIPTION
**Pull request by 100stacks**
Sprint 2: JIRA FRMW-155: https://jira.rax.io/browse/FRMW-155

- [x] Dev LGTM
- [x] Dev LGTM

Migrate `progressbar` docs in README to ngdocs.

### CODE REVIEW

File: src/progressbar/progressbar.js [hunk 1]
```diff
@@ -4,6 +4,32 @@
  * @description
  * # progressbar Component
  *
- * [TBD]
+ * This component may be used to provide feedback on the progress of a workflow or action.
+ *
+ * ## Attributes
```
Comment (CITguy) [inline src/progressbar/progressbar.js]: Attributes and example unnecessary. This is documented in Angular Bootstrap progressbar docs.

File: src/progressbar/progressbar.js [hunk 2]
```diff
@@ -4,6 +4,32 @@
...
+ *   <div class="pure-g columns clear">
+ *     <div class="pure-u-2-3">
+ *       <progressbar class="progress active" max="3" value="3">3 of 3 servers processed
+ *       </progressbar>
+ *     </div>
+ *   </div>
+ * </div>
+ *</pre>
+ *
+ * Encore Framework utilizes Angular Bootstrap *progressbar*. See the
```
Comment (CITguy) [inline src/progressbar/progressbar.js]: Keep this.

File: src/progressbar/progressbar.js [hunk 3]
```diff
@@ -4,6 +4,26 @@
  * @description
  * # progressbar Component
  *
- * [TBD]
+ * This component may be used to provide feedback on the progress of a workflow or action.
+ *
+ * @example
+ * <pre>
+ * <div class="pure-g columns clear">
+ *   <div class="pure-u-2-3">
```
Comment (bry4n) [inline src/progressbar/progressbar.js]: We moved to flex grid from pure-ui framework

use `layout` grid - http://rackerlabs.github.io/encore-ui/#/components/layout

Comment (CITguy) [inline src/progressbar/progressbar.js]: Yes, please.  @100stacks, feel free to ping me if you have any questions.

Comment (100stacks) [inline src/progressbar/progressbar.js]: @CITguy @bry4n should I go ahead and update the `progressbar` component demo as well?

Comment (CITguy) [inline src/progressbar/progressbar.js]: Yes, please.

**Comment by bry4n** [comment]
LGTM

File: src/progressbar/docs/progressbar.html [hunk 4]
```diff
@@ -1,22 +1,22 @@
 <!-- Sample HTML goes here as a live example of how the component can be used -->
 <div ng-controller="progressbarCtrl">
-     <div class="pure-g columns clear">
-        <div class="pure-u-2-3">
+     <div layout="row">
+        <div class="flex-box" flex="67">
```
Comment (bry4n) [inline src/progressbar/docs/progressbar.html]: :+1:

**Comment by 100stacks** [comment]
:ok_hand:

File: src/progressbar/docs/progressbar.html [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
             <progressbar class="progress-striped active" value="45" max="100"></progressbar>
         </div>
     </div>
-     <div class="pure-g columns clear">
-        <div class="pure-u-2-3">
+     <div layout="row">
+        <div class="flex-box" flex="67">
```
Comment (CITguy) [inline src/progressbar/docs/progressbar.html]: Is there a purpose to the `flex-box` class? I'm not sure if it's needed.

Comment (bry4n) [inline src/progressbar/docs/progressbar.html]: I think we can remove `class="flex-box"` from examples

this should work
```html
<div layout="row">
    <div flex> ... </div>
</div>
```

**Comment by 100stacks** [comment]
@bry4n will do.  
```html 
<div flex> ... </div>
```
Does this expand the row?  I was trying to mimic the progessbar length in the demo with `flex="67"`.

**Comment by 100stacks** [comment]
:ok_hand: :+1:

File: src/progressbar/docs/progressbar.html [hunk 6]
```diff
@@ -1,22 +1,22 @@
 <!-- Sample HTML goes here as a live example of how the component can be used -->
```
Comment (CITguy) [inline src/progressbar/docs/progressbar.html]: Actually, this file can be simplified greatly by the following:

```html
<div ng-controller="progressbarCtrl" layout="row">
  <div flex="67">
    <!-- all progressbars go here -->
  </div>
</div>
```

Comment (100stacks) [inline src/progressbar/docs/progressbar.html]: Okay, I will refactor the demo and API docs.

**Comment by 100stacks** [comment]
I just noticed we have additional Pure CSS artifacts in the Styleguide Basics section http://rackerlabs.github.io/encore-ui/#/styleguide/basics#progress-bars -- I will update this demo also.

File: src/progressbar/progressbar.js [hunk 7]
```diff
@@ -4,6 +4,26 @@
...
+ *     </div>
+ *   </div>
+ *   <div layout="row">
+ *     <div flex="67">
+ *       <progressbar class="progress active" max="3" value="3">3 of 3 servers processed
+ *       </progressbar>
+ *     </div>
+ *   </div>
+ * </div>
+ * </pre>
```
Comment (CITguy) [inline src/progressbar/progressbar.js]: I don't think a `<pre>` block example is necessary.  Example usage can be found on the linked Angular Bootstrap page.

Comment (100stacks) [inline src/progressbar/progressbar.js]: Okay, I removed the `<pre>` block.

**Comment by CITguy** [comment]
LGTM

**Pull request merged**